src/main/config/config.c

```diff
@@ -451,8 +451,8 @@ static void resetConf(void)
 
     resetSerialConfig(&masterConfig.serialConfig);
 
-    masterConfig.looptime = 3500;
-    masterConfig.emf_avoidance = 0;
+    masterConfig.looptime = 2000;
```

Comment:
The PR shouldn't change the default looptime since the defaults pids will also need to be changed.